ironfish-cli/src/commands/service/bridge-relay.ts

```diff
@@ -0,0 +1,147 @@
+import { RemoteFlags } from '../../flags'
+
+export default class BridgeRelay extends IronfishCommand {
+  static hidden = true
+
+  static description = `
+    Relay Iron Fish deposits to the Sepolia bridge contract
+  `
+
+  static flags = {
+    ...RemoteFlags,
+    endpoint: Flags.string({
+      char: 'e',
+      description: 'API host to sync to',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_HOST',
+    }),
+    token: Flags.string({
+      char: 't',
+      description: 'API token to authenticate with',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_TOKEN',
+    }),
+    viewKey: Flags.string({
+      char: 'k',
+      description: 'View key to watch transactions with',
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      required: true,
+    }),
+    confirmations: Flags.integer({
+      char: 'c',
+      description: 'Minimum number of block confirmations needed to process deposits',
+      required: false,
+    }),
+  }
+
+  static args = [
+    {
+      name: 'head',
+      required: false,
+      description: 'The block hash to start following at',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { flags, args } = await this.parse(BridgeRelay)
+
+    if (!flags.endpoint) {
+      this.log(
+        `No api host set. You must set IRONFISH_API_HOST env variable or pass --endpoint flag.`,
+      )
+      this.exit(1)
+    }
+
+    if (!flags.token) {
+      this.log(
+        `No api token set. You must set IRONFISH_API_TOKEN env variable or pass --token flag.`,
+      )
+      this.exit(1)
+    }
+
+    const api = new WebApi({ host: flags.endpoint, token: flags.token })
+
+    const confirmations = flags.confirmations ?? this.sdk.config.get('confirmations')
+
+    await this.syncBlocks(api, flags.viewKey, confirmations, args.head)
+  }
+
+  async syncBlocks(
+    api: WebApi,
+    viewKey: string,
+    confirmations: number,
+    head: string | null,
+  ): Promise<void> {
+    this.log('Connecting to node...')
+    const client = await this.sdk.connectRpc()
+
+    this.log('Watching with view key:', viewKey)
+
+    // TODO: track chain state of relay in API
+    if (!head) {
+      const chainInfo = await client.chain.getChainInfo()
+      head = chainInfo.content.genesisBlockIdentifier.hash
+    }
+    this.log(`Starting from head ${head}`)
+
+    const response = client.chain.getTransactionStream({
+      incomingViewKey: viewKey,
+      head,
+    })
+
+    const speed = new Meter()
+    speed.start()
+
+    const buffer = new Array<GetTransactionStreamResponse>()
+
+    for await (const content of response.contentStream()) {
+      if (content.type === 'connected') {
+        buffer.push(content)
+        speed.add(1)
+      } else if (content.type === 'disconnected') {
+        buffer.pop()
+      }
+
+      const committing = buffer.length > confirmations
+
+      this.log(
```

Comment:
This is gonna be a lot of log lines, currently 220k, just fyi

Reply:
yeah this was just copied from the sync command

we can make this a debug-level log or remove it